pkg/runner/local_runner.go

```diff
@@ -74,6 +74,11 @@ func (r *localRunner) Run(ctx context.Context, request *runner.RunRequest) (*run
 	}
 	var cmd *exec.Cmd
 	if r.chrootIntoInputRoot {
+		// The addition of /usr/bin/env is necessary as the runner will run
+		// in a bare container, so the executable won't be found by
```

Comment:
Nit: This isn't necessarily due to the fact that bb-runner is run in a bare container. It's because PATH resolution is done prior to chroot.